pylint_odoo/test_repo/eleven_module/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+{
+    'name': 'Eleven module for tests',
+    'license': 'AGPL-3',
+    'author': u'Jesus, Odoo Community Association (OCA)',
+    'version': '11.0.1.0.0',
```

Comment:
Create the module twelve too, please

Reply:
Done, I created the module twelve_module